addons/website_sale/views/templates.xml

```diff
@@ -1166,7 +1166,7 @@
                         groups="website_sale.group_product_price_comparison"
                         t-options='{
                            "widget": "monetary",
-                           "display_currency": product.currency_id,
+                           "display_currency": website.currency_id,
```

Comment:
Hm, but then, the price is the `product.compare_list_price`, which isn't converted to the website currency ? So the amount isn't correct once you changed the currency. Might be better to also convert this comparison price then. :thinking: